src/cmt_encode_opentelemetry.c

```diff
@@ -64,14 +59,6 @@ static Opentelemetry__Proto__Metrics__V1__MetricsData *
 static void destroy_resource(
     Opentelemetry__Proto__Resource__V1__Resource *resource);
 
-static Opentelemetry__Proto__Resource__V1__Resource *
```

Comment:
All of the code deleted in this file was meant to be used in the next iteration, please revert that change.

Reply:
compiler is warning about an unused function, I think is better to keep it in a separate local branch